common/src/main/java/com/zegelin/cassandra/exporter/CollectorFunctions.java

```diff
@@ -149,31 +158,41 @@ private CollectorFunctions() {}
         };
     }
 
-    public static CollectorFunction<JmxGaugeMBean> histogramGaugeAsSummary() {
-        return histogramGaugeAsSummary(l -> l);
+    public static CollectorFunction<JmxGaugeMBean> histogramGaugeAsSummary(Set<Quantile> excludeQuantiles) {
+        return histogramGaugeAsSummary((l -> l),excludeQuantiles);
     }
 
     /**
      * Collect a {@link SamplingCounting} as a Prometheus summary
      */
-    protected static CollectorFunction<SamplingCounting> samplingAndCountingAsSummary(final FloatFloatFunction quantileScaleFunction) {
+    protected static CollectorFunction<SamplingCounting> samplingAndCountingAsSummary(final FloatFloatFunction quantileScaleFunction,Set<Quantile> excludeQuantiles) {
         return group -> {
             final Stream<SummaryMetricFamily.Summary> summaryStream = group.labeledObjects().entrySet().stream()
                     .map(e -> new Object() {
                         final Labels labels = e.getKey();
                         final SamplingCounting samplingCounting = e.getValue();
                     })
                     .map(e -> {
-                        final Iterable<Interval> quantiles = Iterables.transform(e.samplingCounting.getIntervals(), i -> i.transform(quantileScaleFunction));
-
+                        
+                        Iterator<Interval> itr = e.samplingCounting.getIntervals().iterator();
+                        ArrayList<Interval> filtered = Lists.newArrayList();
```

Comment:
Creating a new list here isn't necessary. Lets just use `Iterables.filter(...)`.

Reply:
As you suggested, I will filter it in TextFormatMetricFormatWriter and revert the above change.